README.md

```diff
@@ -50,6 +54,29 @@ sls deploy
 ```
 5. 👀 Head to the CloudWatch section of the AWS Console to check out your new dashboards 📊 and alarms ⏰ !
 
+
+## Getting Started - CloudFormation Macro
+
+1. Deploying to your account (via the console)
```

Comment:
Can we add an example of how to deploy the SAR app using CloudFormation too?

Reply:
updated cf-macro/README.md with the instructions. Does that look okay?